os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -1148,7 +1147,7 @@ void irq_enable(IRQn_Type IrqNum)
 void irq_disable(IRQn_Type IrqNum)
 {
 	if (IrqNum < 0) {
-		rtw_printf("INT %d should not come here\r\n", IrqNum);
+		DBG_INFO("INT %d should not come here \n", IrqNum);
```

Comment:
Could you let me know why you need a space at the end of the sentence?

Reply:
Just want to separate sentence and the next line "\n", if it is not necessary I can remove it.

Reply:
Let's remove it to reduce unnecessary data in uart send.

Reply:
Sure, will remove it on next push.